en-CA/firefox/browsers/best-browser.ftl

```diff
@@ -0,0 +1,44 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+### URL: https://www-dev.allizom.org//firefox/browsers/best-browser/
+
+best-browser-find-your-best-browser = Find your best browser for speed, privacy and security.
+best-browser-so-many-browser-options = So many browser options, but there’s only one that works best for your needs. The best browser for you should offer both speed and privacy protection.
+best-browser-privacy-speed-and-security = Privacy, speed, and security.
+best-browser-how-to-choose-the-best = How to choose the best browser for you.
+best-browser-the-internet-has-become = The internet has become as essential as electricity and running water, so choosing the best browser for you is more important than ever. The internet is a second office, a teacher and sometimes a medical advisor, even if your actual doctor would prefer you didn’t look up your symptoms online.
+best-browser-in-the-mid-nineties = In the mid-nineties, Netscape, { -brand-name-ie } and AOL dominated the landscape. It was a simpler time when the sweet melody of dial-up internet rang across the land. You learned the meaning of patience waiting for web pages to load. Back then, all that mattered was browser speed.
```

Comment:
`Netscape` shows up multiple times in the [browser history page](https://www-dev.allizom.org/en-US/firefox/browsers/browser-history/). Not sure if it is worth creating an entry. If not, I am fine with that. AOL doesn't appear to have any other mentions in the pages yet to be migrated.